File: Files/MVP.xctemplate/___FILEBASENAME___/View/___FILEBASENAME___ViewController.m
//
//  ___FILENAME___
//  ___PROJECTNAME___
//
//  Created by ___FULLUSERNAME___ on ___DATE___.
//

#import "___VARIABLE_mvpModuleName___ViewController.h"

@implementation ___VARIABLE_mvpModuleName___ViewController

- (void)viewDidLoad
{
    [super viewDidLoad];
    
    [self.presenter viewDidLoad];
}

- (void)viewDidAppear:(BOOL)animated
{
    [super viewDidAppear:animated];
    
    [self.presenter viewDidAppear:animated];
}

@end

File: Files/VPER.xctemplate/___FILEBASENAME___/View/___FILEBASENAME___ViewController.swift
//
//  ___FILENAME___
//  ___PROJECTNAME___
//
//  Created ___FULLUSERNAME___ on ___DATE___.
//

import UIKit

final class ___VARIABLE_productName:identifier___ViewController: UIViewController, ___VARIABLE_productName:identifier___ViewProtocol {

    // MARK: - Properties
    
	var presenter: ___VARIABLE_productName:identifier___PresenterProtocol?
    
    // MARK: - Lifecycle
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        self.presenter?.viewDidLoad()
    }
    
}
